dataplane/freeipa/freeipa.go

```diff
@@ -121,6 +121,40 @@ func assembleRebootRequest(c *cli.Context) *freeIpaModel.RebootInstancesV1Reques
 	}
 }
 
+func RepairFreeIpa(c *cli.Context) {
+	defer commonutils.TimeTrack(time.Now(), "repair FreeIpa cluster")
+	envName := c.String(fl.FlEnvironmentName.Name)
+	RepairInstancesV1Request := assembleRepairRequest(c)
+	freeIpaClient := ClientFreeIpa(*oauth.NewFreeIpaClientFromContext(c)).FreeIpa
+	resp, err := freeIpaClient.V1freeipa.RepairV1(v1freeipa.NewRepairV1Params().WithBody(RepairInstancesV1Request))
+	if err != nil {
+		commonutils.LogErrorAndExit(err)
+	}
+	operationStatus := resp.Payload
+	if operationStatus != nil {
+		log.Infof("[repairFreeIpa] FreeIpa cluster repair requested in environment %s", envName)
+		if operationStatus.OperationID != nil {
+			log.Infof("[repairFreeIpa] Operation ID: %s", *operationStatus.OperationID)
+		}
+		log.Infof("[repairFreeIpa] Status:       %s", operationStatus.Status)
```

Comment:
Info messages don't really appear for end users. Do we want to return this information as other commands do? Either in json or table format?

Reply:
I'm fine if we don't since we're not shipping dp-cli to end users, only used internally.